web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/Forms.js

```diff
@@ -0,0 +1,97 @@
+import { Description } from '../../../components/Description/Description';
+import { Divider } from '../../../components/Divider/Divider';
+import { ErrorWrapper } from '../../../components/Error/Error';
+import { InlineError } from '../../../components/Error/InlineError';
+import { Form, Input, Label, TextArea, Toggle, ToggleRight, Select } from '../../../components/Form';
+import { modal } from '../../../components/Modal/Modal';
+import { EmptyState } from '../../../components/EmptyState/EmptyState';
+import { Tooltip } from '../../../components/Tooltip/Tooltip';
+import { IconEmptyPredictions } from "../../../assets/icons";
+import { useAPI } from '../../../providers/ApiProvider';
+import { ProjectContext } from '../../../providers/ProjectProvider';
+import { Block, cn, Elem } from '../../../utils/bem';
+import { FF_DEV_1682, isFF } from '../../../utils/feature-flags';
+import './MachineLearningSettings.styl';
+
+
+const CustomBackendForm = ({ action, backend, project, onSubmit }) => {
+  const [selectedAuthMethod, setAuthMethod] = useState('');
+  const [mlError, setMLError] = useState();
+  
+  return (
+    <Form action={action}
+          formData={{ ...(backend ?? {}) }}
+          params={{ pk: backend?.id }}
+          onSubmit={async (response) => {
+            if (!response.error_message) {
+              onSubmit(response);
+              /* await fetchBackends(); */
+              /* modalRef.close(); */
+            }
+          }}>
+      <Input type="hidden" name="project" value={project.id}/>
+
+      <Form.Row columnCount={1}>
+        <Input name="title" label="Name" placeholder="Enter a name" required />            
+      </Form.Row>
+
+      <Form.Row columnCount={1}>
+        <Input name="url" label="Backend URL" required />
+      </Form.Row>
+
+      {/* <h4>Authentication Method</h4> */}
+      <Form.Row columnCount={2}>
+        <Select name="auth_method" label="Select authentication method"
+                options={[ { label: "None", value: "NA" }, { label: "Basic Authentication", value: "BA" } ]}
+                onChange={(e) => { setAuthMethod(e.target.value) } } />
+      </Form.Row>
+      
+      {(backend?.auth_method == "BA" || selectedAuthMethod == "BA") && (
+        <Form.Row columnCount={2}>
+          <Input name="basic_auth_user" label="Basic auth user" />
+          <Input name="basic_auth_pass" label="Basic auth pass" type="password" />
+        </Form.Row>
+      )}
+      
+      <Form.Row columnCount={1}>
+        <TextArea name="extra_params" label="Any extra params to pass during model connection" style={{ minHeight: 120 }}/>
+      </Form.Row>
+      
+      <Form.Row columnCount={1}>
+        <div>
+          <Toggle
+            name="is_interactive"
+            label="Interactive preannotations"
+            description="If enabled some labeling tools will send requests to the ML Backend interactively during the annotation process."
+          />
+        </div>
```

Comment:
Is Toggle an inline element? If it is already a block element we can remove the wrapping div:

```suggestion
        <Toggle
          name="is_interactive"
          label="Interactive preannotations"
          description="If enabled some labeling tools will send requests to the ML Backend interactively during the annotation process."
        />
```